stix_shifter_modules/darktrace/stix_transmission/connector.py

```diff
@@ -82,6 +82,8 @@ async def create_results_connection(self, search_id, offset, length):
 
             response_dict = json.loads(response_str)
 
+            response_dict = json.loads(response_wrapper.read().decode('utf-8'))
```

Comment:
`response_dict = json.loads(response_str)` would be better

Reply:
Yeah, just realized that was there twice.